### PR TITLE
Add more common issues to look for in logs

### PR DESCRIPTION
**Pull request by maskersss**
feel free to give feedback on the wording, i didn't really think too much about it so it can likely be improved

### CODE REVIEW

**Review by TheKodeToad — CHANGES_REQUESTED** [review]
Is there any reason you used backticks around certain text? Doesn't that just make it a little harder to read?

Anyway, thank you for the contribution!

**Comment by maskersss** [comment]
i feel like backticks help separate just the regular text from exact button names you should press / an exact java argument / whatever, idk i don't really feel like they make the text harder to read

**Comment by maskersss** [comment]
yeah looking at existing tags they usually explain what to click with more text rather than using backticks and just giving a list of buttons to press:
> To fix this, update forge to the latest version via the Versions tab
> (right click on Forge, click Change Version, and choose the latest one)

rather than

> `Edit > Version > Forge > Change Version > choose the latest one`

, if you want i can rewrite everything in the same style ig

File: src/handlers/event/analyze_logs/issues.rs [hunk 1]
```diff
@@ -271,3 +289,115 @@ fn wrong_java(log: &str) -> Issue {
...
+	let found = log.contains("Could not reserve enough space for ")
+		|| log.contains("Invalid maximum heap size: ");
+	found.then_some(issue)
+}
+
+fn intermediary_mappings(log: &str) -> Issue {
+	let issue = (
+		"Wrong Intermediary Mappings version".to_string(),
+		"You are using Intermediary Mappings for the wrong Minecraft version.
+		Please select the right version in `Edit > Version`."
```
[the remark this answers is not in the document]
Comment (maskersss) [reply src/handlers/event/analyze_logs/issues.rs:383]: would make sense to add backticks around `Change Version` if we're keeping backticks

**Comment by TheKodeToad** [comment]
Since the text is not monospace in the UI I think quotes would be better but that's still quite weird?

**Comment by maskersss** [comment]
> Since the text is not monospace in the UI
wait huh

**Comment by maskersss** [comment]
image
does the `Edit > Version` not look like this to you?

**Comment by TheKodeToad** [comment]
> wait huh? if it's not and it just keeps them as characters then yeah that's not what i intended at all but i was sure backticks just make the text monospace?

The text in the launcher's UI is not monospace, I would personally use them only for something which appears in the logs

**Comment by maskersss** [comment]
ah, i see

**Comment by maskersss** [comment]
removed backticks for everything launcher button related, kept them for java args / refraction tags since thats how it was before

**Review by TheKodeToad — APPROVED** [review]

File: src/handlers/event/analyze_logs/issues.rs [hunk 2]
```diff
@@ -216,12 +225,21 @@ async fn outdated_launcher(log: &str, data: &Data) -> Result<Issue> {
...
 	{
-		let issue = (
-        	"Outdated Prism Launcher".to_string(),
-        	format!("Your installed version is {log_version}, while the newest version is {latest_version}.\nPlease update; for more info see https://prismlauncher.org/download/")
-        );
+		if log_version_parts[0] < 8 {
+			let issue = (
+				"Outdated Prism Launcher".to_string(),
+				format!("Your installed version is {log_version}, while the newest version is {latest_version}.\nPlease update; for more info see https://prismlauncher.org/download/")
+			);
```
Comment (getchoo) [inline src/handlers/event/analyze_logs/issues.rs:232]:
```suggestion
		let issue = if log_version_parts[0] < 8 {
			(
				"Outdated Prism Launcher".to_string(),
				format!("Your installed version is {log_version}, while the newest version is {latest_version}.\nPlease update; for more info see https://prismlauncher.org/download/")
			)
```

File: src/handlers/event/analyze_logs/issues.rs [hunk 3]
```diff
@@ -216,12 +225,21 @@ async fn outdated_launcher(log: &str, data: &Data) -> Result<Issue> {
...
-		Ok(Some(issue))
+			Ok(Some(issue))
+		} else {
+			let issue = (
+				"Outdated Prism Launcher".to_string(),
+				format!("Your installed version is {log_version}, while the newest version is {latest_version}.\nPlease update by pressing the `Update` button in the launcher or using your package manager.")
+			);
+
+			Ok(Some(issue))
+		}
```
Comment (getchoo) [inline src/handlers/event/analyze_logs/issues.rs:242]:
```suggestion
		} else {
			(
				"Outdated Prism Launcher".to_string(),
				format!("Your installed version is {log_version}, while the newest version is {latest_version}.\nPlease update by pressing the `Update` button in the launcher or using your package manager.")
			)
		};
		
        Ok(Some(issue))
```

**Review by getchoo — CHANGES_REQUESTED** [review]

**Review by getchoo — APPROVED** [review]

**Pull request merged**